src/Database/Adapter/Mongo.php

```
@@ -0,0 +1,2329 @@
+     * Set connection and settings
+     *
+     * @param Client $client
+     * @throws MongoException
+     */
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+        $this->client->connect();
+    }
+
+    public function setTimeout(int $milliseconds, string $event = Database::EVENT_ALL): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$event'. (Unused Code Rules)

(UnusedFormalParameter)